crux_kv/README.md

```diff
@@ -4,6 +4,11 @@ This crate contains the `KeyValue` capability, which can be used to ask the Shel
 
 Currently it provides an interface for getting, setting, and deleting keys, checking if keys exists in the store, and listing keys that start with a prefix.
 
+## Getting Started
+
+Add `crux_kv` as a dependency in your app's `Cargo.toml`.
+Add `crux_kv`, with feature `typegen`, as a build dependency in your `share_types` crate's `Cargo.toml`.
```

Comment:
```suggestion
Add `crux_kv`, with feature `typegen`, as a build dependency in your `shared_types` crate's `Cargo.toml`.
```

Reply:
I've since changed this so that we use the re-exported Capability from the shared crate, but with the `typegen` feature enabled.

Reply:
see https://github.com/redbadger/crux/pull/260/commits/316eeb300e0b3f099bb92223549ced607e3d31a1